extensions/trace-propagators/src/main/java/io/opentelemetry/extension/trace/propagation/TraceMultiPropagator.java

```diff
@@ -48,8 +48,12 @@
  * Context context = OpenTelemetry.getPropagators().getTextMapPropagator()
  *   .extract(context, carrier, carrierGetter);
  * }</pre>
+ *
+ * This class is deprecated and will be removed in 0.14.0. Users should migrate to the
```

Comment:
Oops nit - `@deprecated -`

Reply:
I guess that makes it easier to find?  Anyway, added that `@`...